packages/devtools/lib/src/timeline/cpu_profile_protocol.dart

```diff
@@ -12,13 +13,37 @@ import '../utils.dart';
 // debug and profile builds. Do they use different clocks and does this also
 // affect the sampling rate? See https://github.com/dart-lang/sdk/issues/36583.
 
+/// Switch this flag to true collect debug info from the latest cpu profile.
+///
+/// This will add a button to the timeline page that will download the debug
+/// info on click.
+const bool debugCpuProfile = false;
```

Comment:
how can this be modified if it is const?

Reply:
sorry, this was a staged-the-wrong-line-from-git-gui kind of a mistake.